### PR TITLE
Removed mention of Umbraco.tv from the Members Dashboard

### PR DESCRIPTION
**Pull request by sofietoft**
### Prerequisites

- [ ] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes 

### Description

We no longer use Umbraco.tv as the primary source of video material. Instead, we use our Umbraco Learning Base Youtube channel.

I've updated the link on the Members Dashboard.
Not sure if any steps are involved in testing this.

### CODE REVIEW

**Comment by mikecp** [comment]
Hi @sofietoft , thanks for the update 🙂
I see that this text is part of a localized label, `settingsDashboardVideos_trainingDescription`. This means that the text you updated is the "default text" and will only be visible in the languages that do not have a specific translation for this label.

We already had such a case, and the solution we went for, was to create a new label in the en.xml and en_us.xml  file (e.g. "settingsDashboardVideos_learningBaseDescription") and use that one instead. This means that everyone else will get the default text in English again, but at least the links will be correct and can be translated later on in the different language files.

What do you think? Would it be possible for you to do that extra step 😁? 

Cheers!

**Comment by sofietoft** [comment]
Hi, yes! I think it makes sense 🤞 

So, what I'll need to do, is to create the new labels in the language files and add the text there instead - or as well!

Is it correct that this is the place to put the labels:
https://github.com/umbraco/Umbraco-CMS/blob/1db10f3ace8f8508bff11e6e3637d094fc8a200d/src/Umbraco.Core/EmbeddedResources/Lang/en.xml#L343 

Also, should it look something like the description here: 
https://github.com/umbraco/Umbraco-CMS/blob/1db10f3ace8f8508bff11e6e3637d094fc8a200d/src/Umbraco.Core/EmbeddedResources/Lang/en.xml#L2632

**Comment by mikecp** [comment]
You can leave the default text as it is now, and then you need to add it to en.xml and maybe also en-us.xml files

This is the place you are looking for (in en.xml):

image

So you need to create a new key similar to the one with the alias "trainingDescription", with alias like or similar to "learningBaseDescription" and with the adapted content.

**Comment by sofietoft** [comment]
Hi @mikecp !

I've added a new key to en.xml.
Could you verify whether I've done it correctly?

Also, I'm not really sure I understand why I need to create a new key for this and why we couldn't just update/replace content in the old one.
There are no more relevant video content on Umbraco.tv, so we really should not link to that website at all, unless it's directly related to any of the Bridging courses.

**Comment by mikecp** [comment]
Hi @sofietoft,

Thanks for adding the new label in en.xml 👍

Regarding your question, I realize that I was not quite clear in my feedback last time, except maybe in my head 😅

So let me try again 😁

There is the need for a new label, but what I forgot to mention is that this new label has to be used in the file membersdashboardvideos.html instead of the old label.

So, basically, on line 8 in that file, you should use the new label key and have the new default text mentioning Youtube instead of umbraco.tv:

```
<localize key="settingsDashboardVideos_learningBaseDescription>
        <p>Want to master Umbraco? Spend a few minutes learning some best practices by visiting <a href="https://www.youtube.com/c/UmbracoLearningBase" target="_blank" rel="noopener">the Umbraco Learning Base Youtube channel</a>. Here you can find a bunch of video material coverings many aspects of Umbraco.</p>
      </localize>

```

The idea behind this:

- If displaying another language than English, the text inside the "localize" tag is displayed only in case the associated language file does not have a translation for the specified label key.
- So, if we change the text but leave the old label key, all languages that have a translation for that old label will keep showing it and will therefore contain the old links to umbraco.tv from before, So, until someone adapts the old label in the specific language file, it is still the wrong url's that will be shown.
- By using a new label key, we make sure that it does not exist in any language file. This basically means that the content inside the "localize" tag will be shown in all languages. It will be displayed in English for all languages (until someone adds the translation), but the links will be correct.

To summarize, we have to make a trade-off between displaying wrong url's in the user language or displaying the correct url's in English for everyone until translations are added in the language files. And in previous similar situations we decided to go for the second option  😁

I hope my explanation is understandable this time 😅

I can make the extra changes myself if you don't have the time to.

Cheers!

**Review by mikecp — APPROVED** [review]

**Comment by mikecp** [comment]
Hi @sofietoft ,

I felt a little bad about you needing to do some changes again because of my miscommunication, so I took the liberty of making the last modifications myself. I hope you won't mind 😁

Cheers!

**Comment by sofietoft** [comment]
Hi @mikecp !

Thanks for taking care of this one and for explaining (again) 😄 
I've been on vacay for the past 3 three weeks, so haven't been able to follow up.

🙏 
Cheers!
